<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drone System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>840</y>
         <width>151</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="image_label_8">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Chế độ bắn:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="image_label_9">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>--</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="image_label_10">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Đạn:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="image_label_11">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>--</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="image_label_12">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Cửa số trượt:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="selectSlicing">
         <item>
           <property name="text">
            <string>128</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>256</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>512</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>764</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>872</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>940</x>
         <y>0</y>
         <width>116</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="bt_left">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Resources/arrow-102-32.ico</normaloff>Resources/arrow-102-32.ico</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="bt_up">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Resources/arrow-155-32.ico</normaloff>Resources/arrow-155-32.ico</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="bt_down">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Resources/arrow-217-32.ico</normaloff>Resources/arrow-217-32.ico</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="bt_right">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>Resources/arrow-37-32.ico</normaloff>Resources/arrow-37-32.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>840</y>
         <width>241</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="image_label_2">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Tắt toàn bộ camera</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="image_label_5">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="image_label_3">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Góc hướng mạn:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="image_label_7">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>400</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="image_label_6">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Cự ly:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="image_label_14">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Góc hướng toàn phần:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="image_label_15">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="bt_turnoffcamera">
          <property name="text">
           <string>Tắt toàn bộ</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>840</y>
         <width>151</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="1">
         <widget class="QLabel" name="image_label_21">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Bật</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="image_label_17">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>30</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="image_label_18">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Trường nhìn:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="image_label_20">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Ghi lưu:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="image_label_19">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>--</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="image_label_22">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Cự ly:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>940</x>
         <y>120</y>
         <width>112</width>
         <height>217</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="bt_control_file_2">
          <property name="text">
           <string>Zoom +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_control_file_3">
          <property name="text">
           <string>Zoom -</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_control_file_4">
          <property name="text">
           <string>Focus+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_control_file_5">
          <property name="text">
           <string>Focus-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_control_file_6">
          <property name="text">
           <string>Autofocus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_control_file_7">
          <property name="text">
           <string>Đo xa laser</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_control_file_8">
          <property name="text">
           <string>Ổn định con quay</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>bt_control_file_2</zorder>
       <zorder>bt_control_file_5</zorder>
       <zorder>bt_control_file_6</zorder>
       <zorder>bt_control_file_4</zorder>
       <zorder>bt_control_file_8</zorder>
       <zorder>bt_control_file_7</zorder>
       <zorder>bt_control_file_3</zorder>
      </widget>
      <widget class="QFrame" name="frame_6">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>840</y>
         <width>151</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QPushButton" name="bt_video_test">
          <property name="text">
           <string>Chọn video</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="image_label_16">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="text">
           <string>Chế độ video:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="bt_video_thermal">
          <property name="text">
           <string>Camera nhiệt</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="bt_video_main">
          <property name="text">
           <string>Camera màu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="object_list_widget">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>0</y>
         <width>271</width>
         <height>801</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>891</width>
         <height>801</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_1">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 3</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Came 5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 6</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 7</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 8</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">border: 1px solid black;</string>
              </property>
              <property name="text">
               <string>Camera 9</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
